src/nvim/highlight_defs.h

```diff
@@ -114,6 +114,7 @@ typedef enum {
   HLF_WBR,        // Window bars
   HLF_WBRNC,      // Window bars of not-current windows
   HLF_COUNT,      // MUST be the last one
```

Comment:
Well, look at the comment

Reply:
Sorry, all I'm not sure I see the comment you are talking about.

Reply:
`MUST be the last one`

Reply:
Sorry, I'm so dumb,  after messing up the rebase I just made the change manually quickly and forgot to make it second to last.